### PR TITLE
Or83c farnesol detection and reclassification of sensilla

### LINKED ISSUE #206: update synonyms and functions for ORNs
FBrf0224383 ascribes farnesol sensation to Or83c neurons. Also, renames existing ORNs in Table 1 so curent synonyms for adult olfactory receptor neurons where they are named after their receptor, e.g. Or19, might have to be updated to comply with this.

Can GO terms be added to descriptions? E.g. http://amigo.geneontology.org/amigo/term/GO:0097307 for adult olfactory receptor neuron Or83c

### PR DESCRIPTION
**Pull request by Clare72**
In FBrf0224383, as well as showing that Or83c neurons detect farnesol, they also use new sensilla classifications described in FBrf0229748, where two trichoid sensilla are reclassified as intermediate sensilla. This is now reflected in the ontology.
fixes #206

### CODE REVIEW

**Comment by dosumis** [comment]
One small point on this.  This is a pretty clear case of terminogical drift. Trichoid, intermediate and basiconic are *morphological* classifications, so it's a bit odd to reclassify based on *functional* evidence.  However, the boundary between trichoid and intermediate  based on morphology is a bit fuzzy/subjective so I think fine to make the change here.  The fact that the new name/classification is being used in other papers means we should capture in some way.  I would have been more resistant if they'd suggested reclassifying trichoid to basiconic or vice-versa as the morphological difference between these two is clear.

File: fbbt/src/ontology/fbbt-edit.obo [hunk 1]
```diff
@@ -73033,22 +73033,25 @@ creation_date: 2008-05-15T05:24:58Z
 
 [Term]
 id: FBbt:00067000
-name: antennal trichoid sensillum at3
-def: "Olfactory trichoid sensillum of antennal segment 3 that is innervated by 3 ORNs. It is distinguishable from at4 by its odor response profile (de Bruyne and Baker, 2008) and the identity of the ORNs that innervate it (2a, 19a/b and 43a; Couto et al., 2005; de Bruyne and Baker, 2008)." [FlyBase:FBrf0187305, PMID:18535862]
+name: antennal intermediate sensillum ai3
+def: "Olfactory intermediate sensillum of antennal segment 3 that is innervated by 3 ORNs (2a, 19a/b and 43a)." [FBC:CP, FlyBase:FBrf0187305, FlyBase:FBrf0224383, FlyBase:FBrf0229748, PMID:18535862]
+comment: Reclassified from trichoid to intermediate by fluorescence-guided single sensillum recording (FG-SSR) (Lin and Potter, 2015; FBC:CP).
 subset: olfactory_system_draft
+synonym: "antennal trichoid sensillum at3" NARROW []
```
Comment (dosumis) [inline fbbt/src/ontology/fbbt-edit.obo]: Perhaps make this an 'exact' synonym. Both names do refer to the same thing.

Comment (dosumis) [reply fbbt/src/ontology/fbbt-edit.obo]: Or perhaps I'm not following the changes here.  I see the same synonym bellow as exact on at4...

Comment (Clare72) [reply fbbt/src/ontology/fbbt-edit.obo]: Yeah, this paper (FBrf0224383) decided to rename at4 to at3, then two of the same authors subsequently decided that this was a bad idea in FBrf0229748.

File: fbbt/src/ontology/fbbt-edit.obo [hunk 2]
```diff
@@ -73486,23 +73493,24 @@ relationship: part_of FBbt:00067030 ! antennal trichoid sensillum at1
...
 [Term]
 id: FBbt:00067032
-name: antennal trichoid sensillum at2
-def: "Olfactory trichoid sensillum of antennal segment 3 that is innervated by two ORNs." [FlyBase:FBrf0187305, PMID:18535862]
-comment: May be identical to antennal sensillum trichodeum T2 (FBbt:00007348) of Shanbhag et al., 1999.
+name: antennal intermediate sensillum ai2
+def: "Olfactory intermediate sensillum of antennal segment 3 that is innervated by olfactory receptor neuron 83c and olfactory receptor neuron 23a." [FlyBase:FBrf0187305, FlyBase:FBrf0224383, FlyBase:FBrf0229748, PMID:18535862]
+comment: Reclassified from trichoid to intermediate by fluorescence-guided single sensillum recording (FG-SSR) (Lin and Potter, 2015; FBC:CP).
 subset: olfactory_system_draft
+synonym: "antennal trichoid sensillum at2" EXACT []
```
Comment (dosumis) [inline fbbt/src/ontology/fbbt-edit.obo]: Add reference for this synonym if poss.

**Comment by Clare72** [comment]
Synonym refs added - sorry, stuff all over the place in the diff after merging master into branch!

**Comment by dosumis** [comment]
> Synonym refs added - sorry, stuff all over the place in the diff after merging master into branch!

We should probably find time to look at your workflows to see if we can improve this.

**Comment by dosumis** [comment]
Looks good to merge.  Diff prior to re-serialisation is v. hard to read - so please make sure to double check diff for any weirdness after re-serialisation and merge from master.

**Pull request merged**